pyproject.toml

```diff
@@ -17,30 +17,22 @@ authors = [
 license = "MIT"
 
 [tool.poetry.dependencies]
-# ensure compatibility with numba
 python = ">=3.10,<3.13"
-# ensure compatibility with modules compiled with numpy v1.x
 numpy = "*"
-# ensure compatibility with modules compiled with numpy v1.x
-numba = "*"
-# force llvmlite above broken version
+numba = ">=0.60.0"
```

Comment:
This seems to be needed otherwise poetry locks an ancient version of numba, I don't understand why.